src/assets/imgs/theguardian-documentaries.svg

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
How is this .svg being placed in the page?
We should add this through the _mixin file in scss if it's not.